src/Cassandra/Orleans.Clustering.Cassandra/OrleansQueries.cs

```diff
@@ -42,32 +42,33 @@ private OrleansQueries(ISession session)
 
     public ConsistencyLevel MembershipReadConsistencyLevel { get; set; }
 
-    public IStatement EnsureTableExists() => new SimpleStatement("""
-            CREATE TABLE IF NOT EXISTS membership
-            (
-                partition_key ascii,
-                version int static,
-                address ascii,
-                port int,
-                generation int,
-                silo_name text,
-                host_name text,
-                status int,
-                proxy_port int,
-                suspect_times ascii,
-                start_time timestamp,
-                i_am_alive_time timestamp,
+    public IStatement EnsureTableExists(uint defaultTimeToLiveSeconds) => new SimpleStatement(
+        $$"""
+          CREATE TABLE IF NOT EXISTS membership
+          (
+              partition_key ascii,
+              version int static,
+              address ascii,
+              port int,
+              generation int,
+              silo_name text,
+              host_name text,
+              status int,
+              proxy_port int,
+              suspect_times ascii,
+              start_time timestamp,
+              i_am_alive_time timestamp,
 
-                PRIMARY KEY(partition_key, address, port, generation)
-            ) WITH compression = {
-                'class' : 'LZ4Compressor',
-                'enabled' : true
-            };
-            """);
+              PRIMARY KEY(partition_key, address, port, generation)
+          )
+          WITH compression = { 'class' : 'LZ4Compressor', 'enabled' : true }
+            AND default_time_to_live = {{defaultTimeToLiveSeconds}};
```

Comment:
What's the Cassandra behavior if there's an existing table (like from before this PR) but it doesn't have the default TTL configured?
Does it 
* not do anything to the table
* update the table's default TTL for new rows (but leave the existing rows without a TTL set)
* something else

Reply:
It will not do anything to the table